packages/discord.js/src/util/APITypes.js

```diff
@@ -63,6 +63,16 @@
  * @see {@link https://discord-api-types.dev/api/discord-api-types-v10/interface/APIGuild}
  */
 
+/**
+ * @external APIGuildForumDefaultReactionEmoji
+ * @see {@link https://discord-api-types.dev/api/discord-api-types-v10/interface/APIGuildForumDefaultReactionEmoji}
```

Comment:
These don't exist yet 😅 
Can you link me the PR that has them?

Reply:
https://github.com/discordjs/discord-api-types/pull/575